### PR TITLE
[doc] Copyedits to language README

### PR DESCRIPTION
**Pull request by archjayas**
## Motivation

* Copyedits to language README to improve readability.
* Heading updates to ensure consistency with other READMEs.

### Have you read the [Contributing Guidelines on pull requests](https://github.com/libra/libra/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

## Test Plan

Review updated content.

## Related PRs

(If this PR adds or changes functionality, please take some time to update the docs at https://github.com/libra/website, and link to your PR here.)

### CODE REVIEW

File: language/README.md [hunk 1]
```diff
@@ -8,38 +8,38 @@ custom_edit_url: https://github.com/libra/libra/edit/master/language/README.md
...
 * The most complete documentation of the Move IR syntax is the [grammar](https://github.com/libra/libra/blob/master/language/compiler/ir_to_bytecode/src/parser/mod.rs). You can also take a look at the [parser for the Move IR](https://github.com/libra/libra/blob/master/language/compiler/ir_to_bytecode/syntax/src/syntax.lalrpop).
-* Check out the [IR compiler README](https://github.com/libra/libra/blob/master/language/compiler/README.md) for more details on writing Move IR code.
+* Refer to the [IR compiler README](https://github.com/libra/libra/blob/master/language/compiler/README.md) for more details on writing Move IR code.
 
-### Directory Organization
+## How is this module organized?
```
Comment (sblackshear) [inline language/README.md]:
```suggestion
## How is this directory organized?
```

Comment (sblackshear) [reply language/README.md]: I try to avoid using the word "module" to avoid confusion with Move modules.

Comment (archjayas) [reply language/README.md]: Agree, will incorporate that.

Comment (archjayas) [reply language/README.md]: Thanks, I used "folder" instead of module.

**Pull request merged**